lib/options-validator.js

```diff
@@ -69,7 +69,11 @@ module.exports = (pattern, options = {}) => {
                         sizes: Joi.string(),
                         variables: Joi.string(),
                         mixin: Joi.string()
-                    })
+                    }),
+                    callback: Joi.alternatives([
+                        Joi.valid(null),
```

Comment:
I don't think `null` should be valid here, it should either be `undefined` or a function